src/main.js

```diff
@@ -59,6 +61,30 @@ let load = base.loader({
     },
   },
 
+  WorkerState: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      let WorkerState = workerState.setup({
+        account: cfg.azure.account,
+        table: cfg.app.workerStateTableName,
+        credentials: cfg.taskcluster.credentials,
+      });
+      return WorkerState;
+    },
+  },
```

Comment:
I think this may be a merge error?  @jhford just removed the WorkerState entity class.  Awesome that you've rebased your changes against the latest master, by the way!